debian/daliserver.default

```diff
@@ -1,6 +1,8 @@
 # Additional options for daliserver
+# -d <loglevel> sets the logging level (fatal, error, warn, info, default=info)
 # -n enables testing mode without USB connectivity
-# -l sets the listening address (use 0.0.0.0 to listen on all interfaces)
-# -p changes the port
+# -u <bus:dev> only map to one specific USB port (will use the first found otherwise)
```

Comment:
Note that statically configuring bus:dev here is pointless, because these can and will change each time the device is enumerated. The -u option is only useful to pass in bus:dev that have been discovered from the running system.

Reply:
I think it depends. On some systems, the order may be consistent across boots, as long as you don't plug into a different port. But this is only an example I thought may be useful to include here.